### ISSUE TITLE
Unable to create new projects using Nuxt Hub Team

### ISSUE THREAD

**Issue by onmax** [nuxt-hub/core#299]
**Describe the bug**

We are unable to create new projects from the Nuxt Hub within a team.

When we try to create a new project, we get a 400 error:

![image (2)](https://github.com/user-attachments/assets/09b25863-1f90-48c3-8f14-aa8227061ce2)

These are the changes we have made in our team:

- I and another team member are in the team. I am the owner, everything works as expected.

- I am invited to another account in Cloudflare. I get limited access to this new account, so that I can only create pages, not other types of instances.
 
- I change the API token in NuxtHub to connect to the Cloudflare account.
 
- I try to create a new instance, but it fails. At this point I thought it might be due to my limited access in Cloudflare.
 
- I remove the other team member from my Nuxt Hub team.
 
- I add a new team member to the Nuxt Hub team. This member is the same person who controls the account in Cloudflare.
 
- I make him an admin
 
- He tries to initialise the project, but we keep seeing the same error.
 
- We have tried deploying several Nuxt applications but they all seem to return the same error.
 
UPDATED: 

- The admin has tried to create also project using his private account and gives the same error.

---

I am not sure what other context to give or what other information might be important, so please let me know if you want us to try something new.
 

Thanks in advance!

**Comment by atinux** [comment]
Hi @onmax 

What is the error message from the API call?

**Comment by onmax** [comment]
![image](https://github.com/user-attachments/assets/5abec1ff-3203-4105-ba43-01375c5efb95)
![image](https://github.com/user-attachments/assets/fdd3473e-3926-4e25-8ed1-a341de55c967)

**Comment by onmax** [comment]
I have also updated the issue context with the following

> The admin has tried to create also project using his private account and gives the same error.

**Comment by onmax** [comment]
Ok. Now it works. But not sure why it was failing. Here are some reasons:

- In my private Cloudflare account I had projects with the same name. I deleted them.
- When recreating the token in the Cloudflare Dashboard I have the option to "Include" the accounts. we were selecting always the account of my coworker, but I have changed it to "All accounts" and used that token instead. Then in NuxtHub Admin, I selected my coworkers' account.

Closing this issue now since we are able to create projects.

Thank you!
